maps_dashboards/public/components/layer_config/layer_config_panel.tsx

```diff
@@ -74,6 +74,9 @@ export const LayerConfigPanel = ({
   const onUpdate = () => {
     updateLayer();
     closeLayerConfigPanel(false);
+    if (isNewLayer) {
```

Comment:
Why set false when isNewLayer is true?

Should it be ?
```suggestion
    setIsNewLayer(isNewLayer)
```

Reply:
Every time we create new layer, the `isNewLayer ` will be true,  after clicking update, we set it to false. Adding if check here can avoid no need trigger `setIsNewLayer(isNewLayer)` when it's not  a new layer.

Reply:
Thanks for the explanation. Maybe having a common function to unset isNewLayer value to make it easier to understand? 
```
markAsOldLayer() // I cannot come up with a good name here...

markAsOldLayer(() => {
    if(isNewLayer) {
        setIsNewLayer(false);
    }
  });
```
I will leave it to you though as current implementation might be easy enough to someone who is familiar with react.

Reply:
Thanks @heemin32's suggestion , I'll do it later